diff --git a/node_modules/@keystone-6/core/dist/PageContainer-0d8d0d38.esm.js b/node_modules/@keystone-6/core/dist/PageContainer-0d8d0d38.esm.js
index 0363854..1a27272 100644
--- a/node_modules/@keystone-6/core/dist/PageContainer-0d8d0d38.esm.js
+++ b/node_modules/@keystone-6/core/dist/PageContainer-0d8d0d38.esm.js
@@ -1,6 +1,7 @@
 import _extends from '@babel/runtime/helpers/esm/extends';
 import { useTheme, jsx, H3, Text, Stack } from '@keystone-ui/core';
-import { Fragment } from 'react';
+import { Fragment, useState } from 'react';
+import { MenuIcon, XCircleIcon } from '@keystone-ui/icons';
 import { useRouter } from 'next/router';
 import { Button } from '@keystone-ui/button';
 import { Popover } from '@keystone-ui/popover';
@@ -291,25 +292,58 @@ const PageWrapper = props => {
     css: {
       // background: colors.background,
       display: 'grid',
-      gridTemplateColumns: `minmax(300px, 1fr) 4fr`,
-      gridTemplateRows: `${HEADER_HEIGHT}px auto`,
+      gridTemplateColumns: `minmax(300px, 1fr)`,
+      gridTemplateRows: `repeat(2,${HEADER_HEIGHT}px) auto`,
       height: '100vh',
-      isolation: 'isolate'
+      isolation: 'isolate',
+      '@media (min-width: 576px)': {
+        gridTemplateColumns: `minmax(300px, 1fr) 4fr`,
+        gridTemplateRows: `${HEADER_HEIGHT}px auto`
+      }
     }
   }, props)));
 };
-const Sidebar = props => {
+const Sidebar = _ref => {
+  let {
+    isSidebarOpen,
+    ...props
+  } = _ref;
   // const { colors } = useTheme();
 
-  return jsx("aside", _extends({
+  return jsx("div", {
+    css: {
+      gridColumn: '1/2',
+      gridRow: '1/4',
+      display: isSidebarOpen ? 'block' : 'none',
+      height: '100vh',
+      overflowY: 'auto',
+      '@media (min-width: 576px)': {
+        gridColumn: '1/2',
+        gridRow: '2/3',
+        display: 'block',
+        height: '100%'
+      }
+    }
+  }, jsx("div", {
+    onClick: () => {
+      props.setIsSidebarOpen(!isSidebarOpen);
+    },
+    css: {
+      float: 'right',
+      padding: '10px',
+      display: 'block',
+      '@media (min-width: 576px)': {
+        display: 'none'
+      }
+    }
+  }, jsx(XCircleIcon, null)), jsx("aside", _extends({
     css: {
       // borderRight: `1px solid ${colors.border}`,
       minWidth: 0,
       // resolves collapsing issues in children
-      overflowY: 'auto',
       WebkitOverflowScrolling: 'touch'
     }
-  }, props));
+  }, props)));
 };
 const Content = props => {
   const {
@@ -330,16 +364,17 @@ const Content = props => {
     }
   }, props));
 };
-const PageContainer = _ref => {
+const PageContainer = _ref2 => {
   let {
     children,
     header,
     title
-  } = _ref;
+  } = _ref2;
   const {
     colors,
     spacing
   } = useTheme();
+  const [isSidebarOpen, setIsSidebarOpen] = useState(false);
   return jsx(PageWrapper, null, jsx("div", {
     css: {
       alignItems: 'center',
@@ -350,7 +385,17 @@ const PageContainer = _ref => {
       paddingLeft: spacing.xlarge,
       paddingRight: spacing.xlarge
     }
-  }, jsx(Logo, null)), jsx("header", {
+  }, jsx(Logo, null), jsx("div", {
+    onClick: () => {
+      setIsSidebarOpen(!isSidebarOpen);
+    },
+    css: {
+      display: 'block',
+      '@media (min-width: 576px)': {
+        display: 'none'
+      }
+    }
+  }, jsx(MenuIcon, null))), jsx("header", {
     css: {
       alignItems: 'center',
       backgroundColor: colors.background,
@@ -362,7 +407,13 @@ const PageContainer = _ref => {
       paddingLeft: spacing.xlarge,
       paddingRight: spacing.xlarge
     }
-  }, jsx("title", null, title ? `Keystone - ${title}` : 'Keystone'), header), jsx(Sidebar, null, jsx(Navigation, null)), jsx(Content, null, children));
+  }, jsx("title", null, title ? `Keystone - ${title}` : 'Keystone'), jsx("meta", {
+    name: "viewport",
+    content: "width=device-width, initial-scale=1.0"
+  }), header), jsx(Sidebar, {
+    isSidebarOpen: isSidebarOpen,
+    setIsSidebarOpen: setIsSidebarOpen
+  }, jsx(Navigation, null)), jsx(Content, null, children));
 };
 
 export { HEADER_HEIGHT as H, Logo as L, Navigation as N, PageContainer as P, NavigationContainer as a, NavItem as b, ListNavItems as c, ListNavItem as d };
diff --git a/node_modules/@keystone-6/core/dist/common-e0ed4d13.esm.js b/node_modules/@keystone-6/core/dist/common-e0ed4d13.esm.js
index 94011de..c4dd106 100644
--- a/node_modules/@keystone-6/core/dist/common-e0ed4d13.esm.js
+++ b/node_modules/@keystone-6/core/dist/common-e0ed4d13.esm.js
@@ -87,8 +87,12 @@ function ColumnLayout(props) {
       css: {
         alignItems: 'start',
         display: 'grid',
-        gap: spacing.xlarge,
-        gridTemplateColumns: `2fr 1fr`
+        gap: spacing.none,
+        gridTemplateColumns: `100vw`,
+        '@media (min-width: 576px)': {
+          gridTemplateColumns: `2fr 1fr`,
+          gap: spacing.xlarge
+        }
       }
     }, props)))
   );
